docs/examples/obervability/initializer-observability.yaml

```diff
@@ -54,7 +54,8 @@ spec:
           properties:
             chart: grafana
             version: 6.14.1
-            type: helm
+            repoType: helm
```

Comment:
Latest `helm` ComponentDefinition uses the spec filed `repoType` instead of `type` https://github.com/oam-dev/kubevela/blob/master/vela-templates/addons/fluxcd/definitions/helm-release.yaml#L109.